source/taskmanager/src/main/config/queue/priority-task-scheduler.asrm.json

```diff
@@ -1,5 +1,5 @@
 {
-  "workerQueueName": "asrm2",
+  "workerQueueName": "asrm",
```

Comment:
All these `json` files are default configuration to show how you can configure queues, so they don't have to - or even better yet - should not have to match `AERIUS Calculator`. Should we replace them by some basic examples?

Reply:
They can probably be removed as it was initial intended as the source of the configuration files. But that is not needed any more. We can probably remove the whole config directory.